lib/internal/crypto/certificate.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const binding = process.binding('crypto');
+const cryptoUtil = require('internal/crypto/util');
+const toBuf = cryptoUtil.toBuf;
+
+class Certificate {
+  verifySpkac(object) {
+    return binding.certVerifySpkac(object);
+  }
+
+  exportPublicKey(object, encoding) {
+    return binding.certExportPublicKey(toBuf(object, encoding));
+  }
+
+  exportChallenge(object, encoding) {
+    return binding.certExportChallenge(toBuf(object, encoding));
+  }
+}
+
+function createCertificate() {
+  return new Certificate();
+}
+
+cryptoUtil.setHasInstance(createCertificate, Certificate);
+
+module.exports = exports = {
```

Comment:
I've been a little absent from changes to core for the last month. There a reason you're assigning `module.exports` and `exports`?